packages/request-client.js/src/api/request-network.ts

```diff
@@ -136,19 +136,23 @@ export default class RequestNetwork {
       throw new Error(localUtils.formatGetRequestFromIdError(requestAndMeta));
     }
 
+    // get the request state, if nothing confirm take it from pending (useful for the payment network)
```

Comment:
Is it precised to the user (log or something like this) that this state is still pending ?